reco_utils/recommender/ripplenet/model.py

```diff
@@ -0,0 +1,241 @@
+# This code is modified from RippleNet
+# Online code of RippleNet: https://github.com/hwwang55/RippleNet
+
+import tensorflow as tf
+import numpy as np
+from sklearn.metrics import roc_auc_score
+
+class RippleNet(object):
+    def __init__(self, dim, n_hop, kge_weight, l2_weight, lr,
```

Comment:
do you know for some of these parameters a recommended value that we can put as default?

Reply:
here are the original params for movielens and book crossing, maybe i can recommend the ones for movielens? https://github.com/hwwang55/RippleNet/blob/master/src/main.py

Reply:
yeah, that makes sense